src/common/signalk/SKNMEAVisitor.cpp

```diff
@@ -33,13 +33,17 @@
 
 #include "SKNMEAVisitor.h"
 
+// debug only
+#include <KBoxLogging.h>
+
 void SKNMEAVisitor::visitSKElectricalBatteriesVoltage(const SKUpdate& u, const SKPath &p, const SKValue &v) {
   NMEASentenceBuilder sb("II", "XDR", 4);
   sb.setField(1, "V");
   sb.setField(2, v.getNumberValue(), 2);
   sb.setField(3, "V");
   sb.setField(4, p.getIndex());
   _sentences.add(sb.toNMEA() + "\r\n");
+  DEBUG( sb.toNMEA().c_str() );
```

Comment:
Please remove this. If you want to test this, you should just write a test case. It's a much better and faster way to debug these the converters, visitors, etc.